include/bluetooth/hci.h

```diff
@@ -564,6 +564,20 @@ struct bt_hci_rp_write_auth_payload_timeout {
 	uint16_t handle;
 } __packed;
 
+#define BT_DATA_PATH_DIRECTION_INPUT            0x00
+#define BT_DATA_PATH_DIRECTION_OUTPUT           0x01
+#define BT_HCI_OP_CONFIGURE_DATA_PATH           BT_OP(BT_OGF_BASEBAND, 0x0083)
+struct bt_hci_cp_configure_data_path {
+	uint8_t  data_path_direction;
+	uint8_t  data_path_id;
+	uint8_t  vendor_specific_config_len;
```

Comment:
"vendor_specific" is often abbreviated to "vs" in other contexts, I would also suggest to do that here.

Reply:
fixed